src/index.html

```diff
@@ -7,8 +7,76 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <label class="image">
```

Comment:
do not required to have a label, labels are mainly for input fields

Reply:
Understand. Thx. Added all the changes we've been talked about. Still test failed. Waiting for suggestions from you